basicswap/interface/wow.py

```diff
@@ -0,0 +1,574 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2020-2024 tecnovert
+# Distributed under the MIT software license, see the accompanying
+# file LICENSE or http://www.opensource.org/licenses/mit-license.php.
+
+import json
+import logging
+
+import basicswap.contrib.ed25519_fast as edf
+import basicswap.ed25519_fast_util as edu
+import basicswap.util_xmr as xmr_util
+from coincurve.ed25519 import (
+    ed25519_add,
+    ed25519_get_pubkey,
+    ed25519_scalar_add,
+)
+from coincurve.keys import PrivateKey
+from coincurve.dleag import (
+    dleag_prove,
+    dleag_verify,
+    dleag_proof_len,
+    verify_ed25519_point,
+)
+
+from basicswap.interface import (
+    Curves)
+from basicswap.util import (
+    i2b, b2i, b2h,
+    dumpj,
+    ensure,
+    make_int,
+    TemporaryError)
+from basicswap.util.network import (
+    is_private_ip_address)
+from basicswap.rpc_wow import (
+    make_wow_rpc_func,
+    make_wow_rpc2_func)
+from basicswap.chainparams import WOW_COIN, CoinInterface, Coins
+
+
+class WOWInterface(CoinInterface):
+    @staticmethod
+    def curve_type():
+        return Curves.ed25519
+
+    @staticmethod
+    def coin_type():
+        return Coins.WOW
+
+    @staticmethod
+    def COIN():
+        return WOW_COIN
+
+    @staticmethod
+    def exp() -> int:
+        return 11
```

Comment:
@tecnovert is this line for atomic units (decimal places?)
XMR was 12, wownero is 11. Changed to 11. 

please let me know. 
thanks